intro.rst

```diff
@@ -0,0 +1,16 @@
+Introduction
+============
+
+Behaviour Driven Development
+----------------------------
+
+Once you're up and running with Behat, you can learn more about behaviour
+driven development via the following links. Though both tutorials are specific
+to Cucumber, Behat shares a lot with Cucumber and the philosophies are one
+and the same.
```

Comment:
<img width="853" alt="screen shot 2015-08-06 at 12 44 29" src="https://cloud.githubusercontent.com/assets/5813212/9110837/ee4429c0-3c38-11e5-805b-8a0ed04c9c25.png">

There's the side menu for navigating through the documentation?